src/data/randomizer/item_whitelist.h

```diff
@@ -0,0 +1,647 @@
+#include "global.h"
+#include "item.h"
+
+// Randomizer item whitelist
+// Lifted initially from TheXaman's tx_randomizer_and_challenges_branch
+
+static const u16 sRandomizerItemWhitelist[] =
```

Comment:
Thoughts on making a bool in the item struct?

Reply:
If that were done, a table like this would have to be built either at runtime or by preproc. I think that’s a better way to do it, but the preproc hasn’t been written yet.